src/components/title/create/title-create.test.js

```diff
@@ -0,0 +1,173 @@
+import {
+  render,
+  fireEvent,
+} from '@testing-library/react';
+
+import TitleCreate from './title-create';
+
+jest.mock('../_fields/contributor', () => () => (<div>ContributorField component</div>));
+jest.mock('../_fields/edition', () => () => (<div>EditionField component</div>));
+jest.mock('../_fields/publisher-name', () => () => (<div>PublisherNameField component</div>));
+jest.mock('../_fields/publication-type', () => () => (<div>PublicationTypeField component</div>));
+jest.mock('../_fields/identifiers', () => () => (<div>IdentifiersFields component</div>));
+jest.mock('../_fields/description', () => () => (<div>DescriptionField component</div>));
+jest.mock('../_fields/peer-reviewed', () => () => (<div>PeerReviewedField component</div>));
+jest.mock('../_fields/package-select', () => () => (<div>PackageSelectField component</div>));
+jest.mock('../../navigation-modal', () => ({ when }) => (when ? <div>NavigationModal component</div> : null));
+
+const mapMock = jest.fn();
+
+const customPackages = {
+  records: [
+    {
+      id: 'test-package-id1',
+      name: 'test-package-name1',
+    },
+    {
+      id: 'test-package-id2',
+      name: 'test-package-name2',
+    },
+  ],
+  map: mapMock,
+};
+
+const request = {
+  errors: [],
+};
+
+describe('Given TitleCreate', () => {
+  const renderTitleCreate = ({ ...props }) => render(
+    <TitleCreate
+      customPackages={customPackages}
+      onCancel={jest.fn()}
+      onPackageFilter={jest.fn()}
+      onSubmit={jest.fn()}
+      removeCreateRequests={jest.fn()}
```

Comment:
Let's add a `noop` or empty functions `() => {}` for required props functions which will be not affect in tests. In this case we will not handle jest object  for this